client/src/App.jsx

```diff
@@ -17,8 +17,9 @@ class App extends React.Component {
       totalCost: 0,
       sessionComplete: false,
       splitMechanism: '',
-      host: { name: 'host', order: [] }
+      host: { name: 'host', order: [] },
```

Comment:
This might be causing one of the merge conflicts - I don't think we need this anymore since the host is getting added as the first index in the guests array